jekyll/_cci2/security-server.adoc

```diff
@@ -46,7 +46,7 @@ The Audit Log feature is only available for CircleCI installed on your servers o
 
 CircleCI logs important events in the system for audit and forensic analysis purposes. Audit logs are separate from system logs that track performance and network metrics.
 
-Complete Audit logs may be downloaded from the Audit Log page within the Admin section of the application as a CSV file.  Audit log fields with nested data contain JSON blobs.
+Complete Audit logs may be downloaded from the Audit Log page within the Admin section of the application as a CSV file.  Audit log fields with nested data contain JSON blobs. Please note the audit log download can take a very long time to complete; we recommend clicking the Download button once and walking away.
```

Comment:
It might be worthwhile to be more specific. Reading this from a user's perspective, my expectation would be that the download is big and therefore the download takes long. However, what is actually the case is that it takes a long time for the download to even commence. Can we capture that?

Reply:
That's a super good callout, thank you!